packages/components/src/progressCircular/Path.tsx

```diff
@@ -26,15 +27,14 @@ const getPathDescription = (
       a ${pathRadius},${pathRadius} 0 1 1 0,-${2 * pathRadius}
     `;
 
-const getDashStyle = ({
-  dashRatio,
-  pathRadius,
-}: {
+type DashStyleProps = {
   dashRatio: number;
   pathRadius: number;
-}) => {
+  isRtl?: boolean;
+};
+const getDashStyle = ({ dashRatio, pathRadius, isRtl }: DashStyleProps) => {
   const circumference = Math.PI * 2 * pathRadius;
-  const gapLength = (1 - dashRatio) * circumference;
+  const gapLength = (1 - dashRatio) * circumference * (isRtl === true ? -1 : 1);
```

Comment:
as isRtl boolean you can just check it, no needs to do the comparison. It isn't a mistake, just a recommendation